src/test/java/com/serenitydojo/calculator/WhenDoingMaths.java

```diff
@@ -3,48 +3,50 @@
 import org.apache.commons.lang3.StringUtils;
 import org.junit.Test;
 
+import javax.script.ScriptException;
+
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class WhenDoingMaths {
     Calculator calculator = new Calculator();
 
     @Test
-    public void shouldReturnZeroForAnEmptyString() {
+    public void shouldReturnZeroForAnEmptyString() throws Exception {
         assertThat(calculator.evaluate("")).isEqualTo(0);
     }
-
-//    @Test(expected = IllegalMathsOperatorException.class)
-//    public void shouldReportNonSupportedOperations() {
-//        calculator.evaluate("1 ^ 2");
-//    }
-//
-//    @Test
-//    public void shouldReturnTheValueOfASingleNumber() {
-//        assertThat(calculator.evaluate("1")).isEqualTo(1);
-//    }
 //
-//    @Test
-//    public void shouldAddTwoNumbers() {
-//        assertThat(calculator.evaluate("1 + 1")).isEqualTo(2);
-//    }
-//
-//    @Test
-//    public void shouldAddThreeNumbers() {
-//        assertThat(calculator.evaluate("1 + 2 + 3")).isEqualTo(6);
-//    }
-//
-//    @Test
-//    public void shouldAlsoSubtract() {
-//        assertThat(calculator.evaluate("10 - 6")).isEqualTo(4);
-//    }
+    @Test
+    public void shouldReportNonSupportedOperations() throws Exception {
```

Comment:
If you want to check that the exception is thrown, you need to use the "expected" parameter in the @Test annotation 
(see https://github.com/serenity-dojo/string-calculator-challenge/blob/27368a852d6196332b1aabe3bbc14af885930a80/src/test/java/com/serenitydojo/calculator/WhenDoingMaths.java#L16 for an example)

Reply:
I pushed another one, but i still couldn't get it to work.